R/CellGraph.R

```diff
@@ -154,8 +154,8 @@ CreateCellGraphObject <- function (
 #'
 #' edge_list <-
 #'   ReadMPX_item(
-#'     system.file("extdata/PBMC_10_cells",
-#'                 "Sample01_test.pxl",
+#'     system.file("extdata/mock_data",
+#'                 "mock_mpx_data.pxl",
```

Comment:
I think the word "mock" implies a bad quality or that it does not reflect a real case and it should not be taken seriously. But you have done a hard job making a dataset that is exactly the opposite of that! Perhaps we should put a more confident spin on it? E.g. "fiveMPX", "MPX_cells", or something that speaks for its content.

Reply:
Yeah I can change that! I think fiveMPX sounds good.

Reply:
Just pitching in that mock is quite often used in the context of software testing to indicate "not real" which I think it pretty good in this case.

Reply:
Thanks for the inout Johan. I don't have any strong opinions that's a pretty valid point imo. I'm tempted to call it the frankenset ^^

Reply:
The monster is also ok with both:
![image](https://github.com/PixelgenTechnologies/pixelatorR/assets/1788862/9f4813aa-db7d-474e-adb3-ec34ef8cf0b8)

Reply:
Haha! That also sounds good.

Reply:
Yes indeed @johandahlberg  but in this case it is real data although the cells compose a frankenset, hehe. I think "mock" rings of placeholder quality, while this dataset is curated.

I checked available datasets in R `data()` and naming conventions seem to be descriptive and confident:

```
Data sets in package ‘datasets’:

AirPassengers                Monthly Airline Passenger Numbers 1949-1960
BJsales                      Sales Data with Leading Indicator
BJsales.lead (BJsales)       Sales Data with Leading Indicator
BOD                          Biochemical Oxygen Demand
CO2                          Carbon Dioxide Uptake in Grass Plants
ChickWeight                  Weight versus age of chicks on different diets
DNase                        Elisa assay of DNase
EuStockMarkets               Daily Closing Prices of Major European Stock Indices,
                             1991-1998
Formaldehyde                 Determination of Formaldehyde
HairEyeColor                 Hair and Eye Color of Statistics Students
Harman23.cor                 Harman Example 2.3
Harman74.cor                 Harman Example 7.4
Indometh                     Pharmacokinetics of Indomethacin
InsectSprays                 Effectiveness of Insect Sprays
JohnsonJohnson               Quarterly Earnings per Johnson & Johnson Share
LakeHuron                    Level of Lake Huron 1875-1972
LifeCycleSavings             Intercountry Life-Cycle Savings Data
Loblolly                     Growth of Loblolly pine trees
Nile                         Flow of the River Nile
Orange                       Growth of Orange Trees
OrchardSprays                Potency of Orchard Sprays
PlantGrowth                  Results from an Experiment on Plant Growth
```

Reply:
Good point. I agree it's important to stick with language conventions.

Reply:
The data set is now called five_cells. Since it's a just a pxl file, I think it's best if we stick to snake case, but if we create a data set that can be loaded with `data`, we can use a name that better aligns with the names listed above.